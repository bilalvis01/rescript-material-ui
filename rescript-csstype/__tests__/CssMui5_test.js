// Generated by ReScript, PLEASE EDIT WITH CARE
'use strict';

var Curry = require("rescript/lib/js/curry.js");
var React = require("react");
var Mui5$Ress = require("../bindings/Mui5.js");
var Box = require("@mui/material/Box").default;
var ReactTestRenderer = require("react-test-renderer");

test("mui5", (function () {
        var tree = ReactTestRenderer.create(React.createElement(Box, {
                    sx: Curry._1(Mui5$Ress.Css.sx, [
                          Curry._1(Mui5$Ress.Css.color, "blue"),
                          Curry._1(Mui5$Ress.Css.margin, Curry._1(Mui5$Ress.Css.px, 24))
                        ])
                  })).toJSON();
        expect(tree).toMatchSnapshot();
        
      }));

/*  Not a pure module */
