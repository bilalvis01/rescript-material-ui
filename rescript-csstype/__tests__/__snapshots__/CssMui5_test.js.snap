// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mui5 1`] = `
.emotion-0 {
  color: blue;
  margin: 24px;
}

<div
  className="MuiBox-root emotion-0"
/>
`;
